flake-module.nix

```diff
@@ -13,14 +13,16 @@ in
         options.treefmt = mkOption {
           description = ''
             Project-level treefmt configuration
+
+            To access the final treefmt package that uses this configuration,
+            evaluate `config.treefmt.build.wrapper`.
```

Comment:
```suggestion
            Use `config.treefmt.build.wrapper` to get access to the resulting treefmt
            package based on this configuration.
```

Reply:
Also thoughts on https://github.com/numtide/treefmt-nix/pull/29/files

Reply:
Sounds good. Let's add a `defaultFormatter = true;` option that users can set.

Reply:
I think they do support code blocks, it uses markdown.